### DISCUSSION TITLE
Trascr_016: indicazioni "autorità mittente - "atto da trascrivere"

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1036] [Supporto servizi cooperativi]
Il caso d'uso
_Trascr_016 - Trascrizione di nascita: Trascrizione della sentenza di adozione di maggiorenne emessa dal Tribunale_
prevede la presenza della sezione "obbligatoria" della sezione "Autorità mittente":

![image](https://github.com/user-attachments/assets/42748f0f-2d14-4fd0-984e-4eaa333ca03a)

E' corretto ipotizzare che qui debbano essere sempre riportate le informazioni relative al Tribunale che abbia emesso la sentenza di adozione?

Inoltre, si prevede la presenza della sezione "obbligatoria" della sezione "Atto da Trascrivere":
![image](https://github.com/user-attachments/assets/c9d4983a-8206-43db-bb37-c1cacbd40105)

Nella situazione specifica, l'usc dovrebbe occuparsi della trascrizione della sola sentenza di adozione emessa dal Tribunale; è corretto dunque ipotizzare he qui debbano essere sempre riportate le informazioni relative alla sentenza emessa dal Tribunale medesmo? In dettaglio, è corretto che:
- il campo "Data formazione documento" riporti la data di emissione della sentenza di adozione?
- il campo "Estremi documento" riporti l'identificativo della sentenza di adozione?
- il campo "Nome Ente" riporti sempre l'indicazione (fissa) "Tribunale"?

**Comment by acolosimosogei** [comment]
Salve, stiamo lavorando alla discussion #1036, le daremo un riscontro quanto prima.
Grazie.

**Comment by acolosimosogei** [comment]
Salve, le risposte alle domande proposte sono state date in precedenza in una discussion con un caso d'uso differente ma analogo, raggiungibile al seguente link: https://github.com/italia/ansc/discussions/1035

Chiudiamo pertanto la discussion: in caso di ulteriori richieste si prega di aprirne un'altra dedicata.
Grazie
